core/scheduler/src/main/java/Empty.java

```diff
@@ -0,0 +1,5 @@
+public class Empty {
+    // Workaround for this issue https://github.com/akka/akka-grpc/issues/289
+    // Gradle complains about no java sources.
+    // Note. Openwhisk is using a lower gradle version, so the latest akka-grpc version cannot be used.
```

Comment:
Can we fix this? The issue has been fixed

Reply:
I've already tried it, but It requires a higher Gradle version to use the latest akka-grpc.